src/displayapp/screens/Notifications.h

```diff
@@ -69,6 +69,7 @@ namespace Pinetime {
         };
 
       private:
+        DisplayApp* app;
```

Comment:
unused, may be removed from class

Reply:
This is used to call `SetFullRefresh()` function for the dismissal feature.